docs/application/web/guides/connectivity/bluetooth.md

```diff
@@ -295,11 +328,14 @@ To connect to services provided by a server device to the client devices:
 
    When an incoming message is received from the peer device, the `onmessage` event handler in the `BluetoothSocket` interface is triggered.
 
-## Discovering Bluetooth Low Energy Devices
+<!-- TODO #03:
+      - write about method isConnected()
```

Comment:
I would move the section about this method to _Connecting to a Bluetooth Low Energy Device_